src/utils/helpers.ts

```diff
@@ -568,19 +577,36 @@ export function getPercentageChange(oldNumber: number, newNumber: number) {
  * @param timestamp The timestamp that should be converted.
  * @returns A block number.
  */
-export async function timestampToBlock(timestamp: number) {
+export async function timestampToBlock(timestamp: number, network?: string) {
   timestamp =
     String(timestamp).length > 10 ? Math.floor(timestamp / 1000) : timestamp;
+  const apikey = "SSMPT5YB4HKG5NM8Y7SP7Q433GIV94H96U";
```

Comment:
to move into env

Reply:
Can we find another way instead of parsing it into the env file? Since we need to call this on the front-end and vite is not working good with pure env variables and instead use `VITE_...` it is hard to add it as an env.

Reply:
then just pass it from the frontend to here using a `VITE_` env

Reply:
and for the sdk you can declare on the env without `VITE_` since it will only be used for testing and development purposes, so it will be completely separate for different uses